src/com/palmergames/bukkit/towny/TownyFormatter.java

```diff
@@ -378,11 +378,13 @@ private static void getRanks(Town town, List<String> ranklist) {
 
 		if (town.isRuined()) {
 			out.add(Translation.of("msg_time_remaining_before_full_removal", TownRuinSettings.getTownRuinsMaxDurationHours() - TownRuinUtil.getTimeSinceRuining(town)));
-			if (TownRuinSettings.getTownRuinsReclaimEnabled()) {
-				if (TownRuinUtil.getTimeSinceRuining(town) < TownRuinSettings.getTownRuinsMinDurationHours())
+			if (TownRuinSettings.getTownRuinsReclaimEnabled()){
+				if (TownRuinUtil.getTimeSinceRuining(town) < TownRuinSettings.getTownRuinsMinDurationHours()) {
 					out.add(Translation.of("msg_time_until_reclaim_available", TownRuinSettings.getTownRuinsMinDurationHours() - TownRuinUtil.getTimeSinceRuining(town)));
-				else 
+				}
+				else {
 					out.add(Translation.of("msg_reclaim_available"));
+				}

```

Comment:
This can be replaced by the master branch entirely now.

Reply:
Ah, yeah.